easybuild/easyconfigs/c/C-Reduce/C-Reduce-2.8.0-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'C-Reduce'
+version = '2.8.0'
+
+homepage = 'http://embed.cs.utah.edu/creduce/'
+description = """C-Reduce is a tool that takes a large C, C++, or OpenCL
+file that has a property of interest (such as triggering a compiler bug)
+and automatically produces a much smaller C/C++ file that has the same
+property. It is intended for use by people who discover and report bugs
+in compilers and other tools that process source code. C-Reduce is
+released under a BSD-like license."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+source_urls = ['https://github.com/csmith-project/creduce/archive']
+sources = ['creduce-%(version)s.tar.gz']
+checksums = ['ce8f2ec77569073cc93191ac4606894a8731e8c118abb197566a12d8ecd65954']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Perl', '5.28.0'),
+    ('Clang', '6.0.1'),
+    ('Exporter-Lite', '0.08', '-Perl-%(perlver)s'),
+    ('Getopt-Tabular', '0.3', '-Perl-%(perlver)s'),
```

Comment:
@akesandgren Wouldn't it make more sense to include these as extensions, to avoid having stand-alone modules for them?

Or to include them in the existing `Perl` easyconfig as extensions?

Reply:
Maybe, but i was aiming for the easy way out here just to get something that works.
And i did not want to update the existing Perl, maybe some newer release, but not the current one.
I can look at including them as extensions but it will, probably, be a while before i have time.